src/main/java/blackjack/Application.java

```diff
@@ -0,0 +1,108 @@
+
+import blackjack.domain.Round;
+import blackjack.domain.WinningDiscriminator;
+import blackjack.domain.card.Card;
+import blackjack.domain.card.CardDeck;
+import blackjack.domain.card.CardShape;
+import blackjack.domain.card.CardShuffler;
+import blackjack.domain.card.CardType;
+import blackjack.domain.gambler.Name;
+import java.util.Arrays;
+import java.util.List;
+
+public class Application {
+    public static void main(String[] args) {
+        CardDeck cardDeck = createCardDeck();
+        List<Name> playerNames = getPlayerNames();
+        Round round = new Round(cardDeck, playerNames);
+        round.distributeInitialCards();
+        printInitialDistributionPrompt(playerNames);
+        printInitialCards(round, playerNames);
+        processPlayersTurn(playerNames, round);
+        processDealerTurn(round);
+        printGameResult(round, playerNames);
+    }
+
+    private static CardDeck createCardDeck() {
+        List<Card> cards = Arrays.stream(CardShape.values())
+                .flatMap(shape -> Arrays.stream(CardType.values())
+                        .map(type -> new Card(shape, type)))
+                .collect(toList());
+        return new CardDeck(cards, new CardShuffler());
+    }
+
+    private static List<Name> getPlayerNames() {
+        try {
+            return inputPlayerName();
+        } catch (IllegalArgumentException e) {
+            System.out.println(e.getMessage());
+            return getPlayerNames();
+        }
+    }
+
+    private static void printInitialCards(final Round round, final List<Name> playerNames) {
+        printGamblerCards(Name.getDealerName(), round.getInitialCards(Name.getDealerName()));
+        for (Name playerName : playerNames) {
+            printGamblerCards(playerName, round.getInitialCards(playerName));
+        }
+    }
+
+    private static void processPlayersTurn(final List<Name> playerNames, final Round round) {
+        for (Name playerName : playerNames) {
+            processPlayerTurn(round, playerName);
+        }
+    }
+
+    private static void processDealerTurn(final Round round) {
+        if (round.isGamblerCanReceiveCard(Name.getDealerName(), Round.DEALER_RECEIVE_CRITERIA)) {
+            round.distributeCards(Name.getDealerName(), 1);
+            printDealerReceiveCard();
+        }
+    }
+
+    private static void processPlayerTurn(final Round round, final Name playerName) {
+        while (isHit(playerName)) {
+            round.distributeCards(playerName, 1);
+            printGamblerCards(playerName, round.getCards(playerName));
+            boolean isBusted = !round.isGamblerCanReceiveCard(playerName, WinningDiscriminator.BLACK_JACK);
+            if (isBusted) {
+                printBusted(playerName);
+                break;
+            }
+        }
+        if (!round.isPlayerOwnsCardExceptInitialCards(playerName)) { // 첫 카드를 받은 이후로 카드를 받지 않은 경우
+            printGamblerCards(playerName, round.getCards(playerName));
+        }
```

Comment:
```suggestion
        while (round.isGamblerCanReceiveCard(playerName, WinningDiscriminator.BLACK_JACK) && isHit(playerName)) {
            round.distributeCards(playerName, 1);
            printGamblerCards(playerName, round.getCards(playerName));
        }

        if (!round.isGamblerCanReceiveCard(playerName, WinningDiscriminator.BLACK_JACK)) {
            printBusted(playerName);
        }
        if (!round.isPlayerOwnsCardExceptInitialCards(playerName)) { // 첫 카드를 받은 이후로 카드를 받지 않은 경우
            printGamblerCards(playerName, round.getCards(playerName));
        }
```

이렇게도 가능할 거 같네요. while 문 안에 있는 if문을 제거해 뎁스를 줄일 수 있을 것 같습니다. 주석으로 `첫 카드를 받은 이후로 카드를 받지 않은 경우`를 명시해두셨는데, 이미 메서드로 의미를 잘 나타내고 있지 않나요?? 어떤 점이 마음에 걸리셔서 주석을 추가하셨을까요?